lib/src/components/motor/motor.dart

```diff
@@ -33,22 +37,71 @@ abstract class Motor extends Resource {
 
   Motor(this.name);
 
+  /// Sets the "percentage" of power the [Motor] should employ between -1 and 1.
+  /// When [powerPct] is negative, the rotation will be in the backward direction.
+  ///
+  /// Args:
+  ///     [powerPct] (double): Power between -1 and 1
+  ///         (negative implies backwards).
   Future<void> setPower(double powerPct, {Map<String, dynamic>? extra});
 
+  /// Spin the [Motor] the specified number of [revolutions] at specified [rpm].
+  /// When [rpm] or [revolutions] is a negative value, the rotation will be in the backward direction.
+  /// Note: if both [rpm] and [revolutions] are negative, the motor will spin in the forward direction.
+  ///
+  /// Args:
+  ///     [rpm] (double): Speed at which the motor should move in rotations per minute
+  ///         (negative implies backwards).
+  ///     [revolutions] (double): Number of revolutions the motor should run for
+  ///         (negative implies backwards).
   Future<void> goFor(double rpm, double revolutions, {Map<String, dynamic>? extra});
 
+  /// Spin the [Motor] to the specified position (provided in revolutions from home/zero),
+  /// at the specified speed, in revolutions per minute.
+  /// Regardless of the directionality of the [rpm] this function will move
+  /// the [Motor] towards the specified position.
+  ///
+  /// Args:
+  ///     [rpm] (double): Speed at which the motor should rotate (absolute value).
+  ///     [positionRevolutions] (double): Target position relative to home/zero, in revolutions.
```

Comment:
It's actually not super important to have these enumerated like this, because the flutter documentation generator is much better than the python one.

So while in python it's necessary to list out all args and return types, it actually looks bad in Flutter to do so:
> ![image](https://user-images.githubusercontent.com/2429728/236062126-e59d04c9-ea6e-4704-b3ad-bd2e5d3bdb7d.png)

You can see that it doesn't format the args nicely and it just looks weird. 

Unlike in python where it formats like this:
> ![image](https://user-images.githubusercontent.com/2429728/236062311-dee3319a-d3e3-4cb3-b274-2b0caf6075b6.png)

So I would remove all the `Args` and `Returns` lines.

Reply:
Thanks for calling this out. I will remove those.

Reply:
✅ removed the Args and Returns from the documentation.